packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7892,9 +7895,13 @@ export default class Meeting extends StatelessWebexPlugin {
    * Gets the time left in seconds till the permission token expires
    * (from the time the function has been fired)
    *
-   * @returns {number} time left in seconds
+   * @returns {object} containing permissionTokenTimeLeft, permissionExpiry, currentTime
    */
-  public getPermissionTokenTimeLeftInSec(): number | undefined {
+  public getPermissionTokenTimeLeftInSec(): {
+    permissionTokenTimeLeft: number | undefined;
```

Comment:
just FYI, you can do `permissionTokenTimeLeft?: number` - it's shorter than doing `| undefined`